core/documentation/introduction/Welcome.mdx

```diff
@@ -1,8 +1,21 @@
 import { Meta } from "@storybook/blocks";
+import json from "../../../package.json";
 
-<Meta title="Documentation/Introduction/Introduction" />
+<Meta title="Documentation/Introduction/Welcome" />
 
-# Introduction
+<div class="full-width-bg">
+# Core Style Library
+
+</div>
+
+<div class="version-badges">
+  <span class="fudis-badge fudis-badge__primary">
+    [NPM v{json.version}](https://www.npmjs.com/package/@funidata/fudis-core)
+  </span>
+  <span class="fudis-badge fudis-badge__primary">
+    [Core Github](https://github.com/funidata/fudis-core)
+  </span>
+</div>
```

Comment:
In addition to these two badges I would also add the LICENCE badge, which would redirect to GitHub https://github.com/funidata/fudis/blob/main/LICENSE